python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -668,7 +685,7 @@ def process_graph(args):
             if args.num_processes_for_nodes is not None else args.num_processes
     num_processes_for_edges = args.num_processes_for_edges \
             if args.num_processes_for_edges is not None else args.num_processes
-    verify_confs(process_confs)
+    verify_confs(process_confs, args.add_reverse_edges)
```

Comment:
How to handle the case if a user wants to add reverse edges?
We should have some thing like
```
etype = (th.cat((src, dst), th.cat(dst, src))
```
And also if there is edge features, we should at least crash the gconstruct.